### ISSUE TITLE
LG 1-8 Translation inconsistency

### ISSUE THREAD

**Issue by ulibecker** [isaqb-org/curriculum-foundation#378]
DE: Explizite von impliziten Aussagen unterscheiden
EN: Distinguish Explicit Statements and Implicit Assumptions

I prefer the current english version, therefore propose to change the german version to "Explizite Aussagen von impliziten Annahmen unterscheiden"

**Comment by gernotstarke** [comment]
Good idea, let's update and create a new release...Regards, Gernot Dr. Gernot ***@***.*** www.gernotstarke.deVon einem mobilen Geraet gesendet - bitte verzeihen Sie eventuelle Tippfehler. Am 28.03.2023 um 14:56 schrieb Ulrich Becker ***@***.***>:﻿
DE: Explizite von impliziten Aussagen unterscheiden
EN: Distinguish Explicit Statements and Implicit Assumptions
I prefer the current english version, therefore propose to change the german version to "Explizite Aussagen von impliziten Annahmen unterscheiden"

**Comment by gernotstarke** [comment]
hhm - I just noticed it's still not as clear as it could be:

DE (is):  "explizite von impliziten Aussagen unterscheiden

DE (should be): explizite Aussagen von implizten Annahmen unterscheiden

The change did not fix this
